src/bluecoins_cli/adb.py

```diff
@@ -62,10 +62,17 @@ def start_app(self, activity: str) -> None:
         self.device.app_start(APP_ID, activity)
 
 
-device = Device.connect()
-device.stop_app()
-db = get_db_name()
-device.pull_db(db)
-device.cli_command_run('convert', db)
-device.push_db_root(db)
-device.start_app('.ui.activities.main.MainActivity')
+def execute_cli_command_with_adb(cli_command: str, activity: str, keys_value: str = '') -> None:
```

Comment:
It's unobvious what's `activity` argument purpose (to start after the final sync is complete). Also, optional?